### DISCUSSION TITLE
Booking Email Recipients Determined at Queued time and not Planned time

### DISCUSSION THREAD

**Discussion by roysinn** [liedekef/events-made-easy#292] [Q&A] [answered]
Just confirming - when a Booking email is scheduled for a future date, the recipient list is determined at the time the planned email was queued and not when the email is actually sent. In other words, after a Booking email is queued, any new registrations are not added to the recipient list for that planned mailing. Is that correct? 

If so, is any way in EME to queue a booking email so that the recipient list is deterred at mailing?

Thanks in advance.

**Comment by liedekef** [comment, marked as answer]
While the recipients are evaluated at queue time, they are re-evaluated at send time to make sure the correct people get it.

**Reply by roysinn** [reply]
Yes! Thank you again for such a well designed system.

**Reply by liedekef** [reply]
The next version will add a warning in the mailer overview for planned mails, so this will be clear to everybody.
